docs/API.md

```diff
@@ -171,7 +176,17 @@ curl -v \
 ```json
 {
   "id": "81625c14128d46c2b600e74a017fa4a8",
-  "url": "https://secure.gravatar.com/avatar/6d940dd41e636cc156074109b8092f96"
+  "avatar": "https://secure.gravatar.com/avatar/6d940dd41e636cc156074109b8092f96"
```

Comment:
Since we don't support gravatar, let's change the example text to a plausible URL.  Also, was the doc here in error with "url" vs "avatar"?